### PR TITLE
Add unban_pubkey method

### PR DESCRIPTION
**Pull request by smk762**

### CODE REVIEW

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# unban\_pubkey
```
Comment (artemii235) [inline docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md:1]: The method is `unban_pubkeyS`

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+**unban_pubkey type data**
+
+The `unban_pubkey` method will remove all currently pubkeys from your ban list, or specific pubkeys from a user defined list.
```
Comment (artemii235) [inline docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md:5]: It seems that "banned" is missing in "currently pubkeys".

**Review by artemii235 — CHANGES_REQUESTED** [review]
Thanks for the PR! I have few changes requests.

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md (hunk 2 above)
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/unban_pubkey.md]: fixed, thanks.

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# unban\_pubkeys
+
+**unban_pubkey type data**
```
Comment (artemii235) [inline docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md:3]: It is still `unban_pubkey` 🙂

**Review by artemii235 — CHANGES_REQUESTED** [review]
One more note

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md (hunk 3 above)
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md]: thanks, fixed - excuse my blindness!

**Review by artemii235 — APPROVED** [review]
👍

File: docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md (hunk 3 above)
Comment (artemii235) [reply docs/basic-docs/atomicdex-api-legacy/unban_pubkeys.md]: It's NP 🙂

**Review by gcharang — APPROVED** [review]

**Pull request merged**